src/modules/pos/validator/components/ClaimRewardsForm/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useRewardsClaimable } from '@pos/reward/hooks/queries';
+import classNames from 'classnames';
+import BoxHeader from '@theme/box/header';
+import { useTranslation } from 'react-i18next';
+import { MODULE_COMMANDS_NAME_MAP } from '@transaction/configuration/moduleCommand';
+import TxComposer from '@transaction/components/TxComposer';
+import { useCurrentAccount } from '@account/hooks';
+import { QueryTable } from '@theme/QueryTable';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import styles from './ClaimRewardsForm.css';
+
+const rewardsClaimableHeader = (t) => [
+  {
+    title: t && t('Token'),
+    classList: `${grid['col-xs-3']}`,
+  },
+  {
+    title: t && t('Reward amount'),
+    classList: `${grid['col-xs-4']}`,
+  },
+  {
+    title: t && t('Fiat'),
+    classList: `${grid['col-xs-1']}`,
+  },
+];
```

Comment:
Also I would suggest that this is also abstracted to a different file just as we did for the rest of our tables we can all this new file lets say `rewardsClaimableHeaderMap`.

Reply:
Also separate file now